source/util/FileUtils.cpp

```diff
@@ -302,3 +302,26 @@ bool FileUtils::FileExists(const string &filename)
     ifstream f(expandedPath);
     return f.good();
 }
```

Comment:
Add another function to validate if the file path does not have any illegal filePath characters.